languages/java/app-encryption/pom.xml

```diff
@@ -7,6 +7,34 @@
   <groupId>com.godaddy.asherah</groupId>
   <artifactId>appencryption</artifactId>
   <version>1.0.0-SNAPSHOT</version>
+  <name>Asherah</name>
+  <description>
+      An application-layer encryption SDK that provides advanced encryption features and in depth defense against
+      compromise.
+  </description>
+  <url>https://github.com/godaddy/asherah</url>
+
+  <licenses>
+      <license>
```

Comment:
match the indent level in this file (for all changes in this file).

side note: we should probably do a sweep in other POMs and the new `settings.xml` to update/normalize to same indent level (probably better in separate PR). i know AE and SM have always been different (we kept forgetting to fix). pretty sure the maven XML file standard is 2 spaces (also preferred for not nesting too deeply IMO)